### PR TITLE
Deploy to Production

### PR DESCRIPTION
**Pull request by cesarvarela**
//

### CODE REVIEW

File: site/gatsby-site/migrations/2023.05.15T11.49.00.csetv1-remove-ai-qualifier.js [hunk 1]
```diff
@@ -0,0 +1,2033 @@
...
+
+  for (const classification of classifications) {
+    classificationsCollection.updateOne(
+      { namespace: classification.namespace, incident_id: classification.incident_id },
+      {
+        $set: {
+          attributes: classification.attributes.map((attribute) => {
+            let short_name = attribute.short_name;
+
+            if (short_name == 'Clear link to AI') {
```
Comment (cesarvarela) [inline site/gatsby-site/migrations/2023.05.15T11.49.00.csetv1-remove-ai-qualifier.js:37]: @lmcnulty looks like this change is breaking this test https://app.netlify.com/sites/staging-aiid/deploys/648923203cd16b0008c3dfbd#L532 (which I rewritten for clarity) .

Does the taxonomy definition need to be updated too?

Comment (cesarvarela) [reply site/gatsby-site/migrations/2023.05.15T11.49.00.csetv1-remove-ai-qualifier.js:37]: I just noticed that one is `technology` and another is `Technology` 🤔

**Comment by cesarvarela** [comment]
@smcgregor ready for review, 

Includes a fix for #2088, (the Item needs to be deleted, and added again, and will be saved properly)

**Comment by smcgregor** [comment]
Layout changes really improve the responsiveness. Hydration error is annoying although it was not introduced with this. Merging.

**Pull request merged**